vars/dockerBuildArgs.groovy

```diff
@@ -20,13 +25,19 @@ int getuid() {
 String call(Map config = [:]) {
     Boolean cachebust = true
     Boolean add_repos = true
-    if (config['cachebust']) {
+    Boolean deps_build = false
+    if (config.containsKey('cachebust')) {
```

Comment:
Just wondering what the difference in syntax is here.  My understanding is that an unset key is false in groovy.

Reply:
It is.  The default is true so if you set it to false by a parameter then it doesn't update the default so it remained false.